atom/utility/atom_content_utility_client.cc

```diff
@@ -101,6 +101,7 @@ bool AtomContentUtilityClient::OnMessageReceived(
 
   bool handled = true;
   IPC_BEGIN_MESSAGE_MAP(AtomContentUtilityClient, message)
+    IPC_MESSAGE_HANDLER(BraveUtilityMsg_GetImportFromChrome, OnImportFromChrome)
```

Comment:
this doesn't seem necessary. The `SourceProfile` is sent to `atom::ExternalProcessImporterClient` and we can set the command line switch there